vuetifyjs/vuetify
# fix(sass variables): add missing !default suffixes to numerous variables

## Description
Suffixed !default to few SASS variables of VAutocomplete, VBadge, VCalendar, VExpansionPanel and VMain so that these variables can be overwritten in a custom theme. Fixes #11751 

## How Has This Been Tested
1. Import a custom theme where VAutocomplete SASS variable **$autocomplete-focused-input** has been overwritten.
1. Use the component and check whether the input CSS has indeed changed.

<details>
```Vuetify.js
import Vue from 'vue'
import Vuetify from 'vuetify'
import * as locales from '../src/locale'
import { preset } from "vue-cli-plugin-vuetify-preset-responsive/preset"

Vue.use(Vuetify)

export default new Vuetify({
  preset,
  lang: {
    locales,
  },
})
```


```vue-playground
<v-autocomplete outlined :items="[1, 2, 3]"></v-autocomplete>
```
<details>

## Types of changes
[x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
[x] The PR title is no longer than 64 characters.
[x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
[x] My code follows the code style of this project.
[ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
